pkg/limitador/k8s_objects.go

```diff
@@ -216,22 +207,50 @@ func Labels(limitador *limitadorv1alpha1.Limitador) map[string]string {
 	}
 }
 
-func deploymentContainerCommand(storage *limitadorv1alpha1.Storage, storageConfigSecret *v1.Secret, rateLimitHeaders *limitadorv1alpha1.RateLimitHeadersType) []string {
-	command := []string{"limitador-server"}
-
-	// stick to the same default as Limitador
-	if rateLimitHeaders != nil {
-		command = append(command, "--rate-limit-headers", string(*rateLimitHeaders))
+func PVC(limitador *limitadorv1alpha1.Limitador) *v1.PersistentVolumeClaim {
+	pvc := &v1.PersistentVolumeClaim{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "PersistentVolumeClaim",
+			APIVersion: "v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      PVCName(limitador),
+			Namespace: limitador.ObjectMeta.Namespace,
+			Labels:    Labels(limitador),
+		},
+		Spec: v1.PersistentVolumeClaimSpec{
+			AccessModes: []v1.PersistentVolumeAccessMode{
+				v1.PersistentVolumeAccessMode("ReadWriteOnce"),
```

Comment:
What if multiple replicas?

Reply:
According to [k8s docs on PV Access Modes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes),
```
ReadWriteOnce
    the volume can be mounted as read-write by a single node. ReadWriteOnce access mode still can allow multiple pods to access the volume when the pods are running on the same node.
```
So, there is a limitation there. `ReadWriteMany` mode requires specific cluster infra like NFS storage class usually not in place in clusters.

Reply:
@alexsnaps Let's say multiple limitador replicas (in the same node) have R/W access to the same path. [RocksDB FAQ](https://github.com/facebook/rocksdb/wiki/RocksDB-FAQ#basic-readwrite) is quite clear, but I want to double check with you. 

```
Q: Can I write to RocksDB using multiple processes?

A: No. However, it can be opened using Secondary DB. If no write goes to the database, it can be opened in read-only mode from multiple processes.
```
The limitador operator definitely needs to ensure multiple replicas are not being deployed when using disk storage. Right? (By the way, we have the same issue with in memory deployment)

Reply:
👍 local disk storage would be a single instance deployment (i.e. no replicas)